Backend/src/main/resources/application.properties

```diff
@@ -1,15 +1,15 @@
-server.port=9090
+server.port= 9090 
 spring.jpa.database=POSTGRESQL
 spring.datasource.platform=postgres
 spring.datasource.url=jdbc:postgresql://localhost:5432/postgres
 spring.datasource.username=postgres
 spring.datasource.password=postgres
 spring.jpa.show-sql=true
-spring.jpa.generate-ddl=true
-spring.jpa.hibernate.ddl-auto=update
+spring.jpa.generate-ddl=true 
+spring.jpa.hibernate.ddl-auto=create-drop
```

Comment:
why is it create-drop? it should be update as we dont want to lose data on restart

Reply:
Default config, the user can change it!